src/main/java/com/google/step/servlets/PromoCodeServlet.java

```diff
@@ -38,21 +38,26 @@ public void init() {
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String channelId = request.getParameter("formInput");
```

Comment:
We should change the parameter name to channelURL everywhere to be more specific about what the parameter is.